device-types/Avocent/ACS8048.yaml

```diff
@@ -1,7 +1,8 @@
 ---
 manufacturer: Avocent
-model: ACS8048
+model: ACS 8048
 slug: acs8048
+part_number: ACS-8048
```

Comment:
Please add a comment in markdown format if you have a link to the manufacturer provided datasheet

Reply:
There are several models of type 8048, so I refined the definition of one of the model + added link to the description